services/network/main.py

```diff
@@ -91,13 +93,32 @@ def get_saved_wifi_network(self):
 
         return WifiManager.__dict_from_table(data)
 
+    def hash_password(self, ssid, password):
```

Comment:
should be staticmethod

Reply:
done

Reply:
wait, can you run a static method with self ? you should remove the self parameter.

Reply:
oops done thanks